cmd/ftl/cmd_serve.go

```diff
@@ -105,6 +118,66 @@ func (s *serveCmd) Run(ctx context.Context) error {
 	return nil
 }
 
+func setupDB(ctx context.Context) (*string, error) {
+	logger := log.FromContext(ctx)
+
+	dsn := fmt.Sprintf("postgres://postgres:secret@localhost:5433/%s?sslmode=disable", ftlDBName)
```

Comment:
I think it would be a better idea to dynamically allocate the port, then retrieve it, rather than hard-coding it. Either that, or add a flag for controlling it.